site/docs/v1/tech/apis/identity-providers/_links-post-response-body.adoc

```diff
@@ -1,6 +1,9 @@
 ==== Response Body
 
 [.api]
+[field]#identityProviderLink.displayName# [type]#[String]#::
+A human readable name for this link for identification and convenience. This value will generally be an email address, or username. This value can be set by the API caller. So for instance, this value could be `Initial piedpiper link from user migration`. This value will be updated during the next login for the linked user based on the identify provider (`richard@piedpiper.com`, for instance). This value may not necessarily reflect the username or email you used to authenticate with the 3rd party identity provider.
+
```

Comment:
This is the description for the response. It may be better to just describe how the value may be modified by a future login to this IdP on the request side.

For example, the request could say:

> A human readable name for this link. This value should be used to make it easier to identify the user this link represents in the remote identity provider. 
>
> This value is optional and it will always be set by FusionAuth the next time this link is used to resolve the FusionAuth user during a login event for this IdP.

And the response could be:

> A human readable name for this link. In most cases this value will be the username or email address of the user in the remote identity provider.

Reply:
ok, that makes sense.